package.json

```diff
@@ -17,11 +17,12 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "lint": "eslint ./src --ext js,ts,tsx",
-    "coverage": "react-scripts test --coverage --reporter=html",
+    "start": "env-cmd -f .env.development react-scripts start",
+    "build_preprod": "env-cmd -f .env.preproduction react-scripts build",
+    "build_prod": "env-cmd -f .env.production react-scripts build",
+    "test": "env-cmd -f .env.development react-scripts test",
+    "lint": "eslint \"./src/**/*.ts[x]\"",
+    "coverage": "env-cmd -f .env.development react-scripts test --coverage --reporter=html",
```

Comment:
```suggestion
    "coverage": "npm test -- --coverage --reporter=html",
```

Reply:
> Great work. I only added one suggestion to make future changes to the `test` script easier.

I took in consideration your changes request. Moreover, I added `env-cmd` missing package used in commands.